BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -412,7 +412,7 @@ extension WalletViewController {
   func startNotificationObserver() {
     // Stopping as a precaution
```

Comment:
these comments are not needed anymore I guess

Reply:
Yeah I don think that was ever needed, probably just missed in PR